docs/modules/ROOT/pages/explanations/decisions/redis_galera.adoc

```diff
@@ -0,0 +1,64 @@
+= Redis and MariaDB/Galera Provisioning
+
+This page combines the decisions for deploying Redis, MariaDB and Galera, as they are already in use for SPKS.
+
+== Problem
+
+We want to re-use proven methods to deploy Redis and MariaDB/Galera.
+
+== Operators
+
+For Redis as well as MariaDB/Galera there are a few operators available.
+But most of those are in a very early development stage.
+Or don't support the applications we want to provide.
+
+The following section contains the operators that have been looked at.
+We might re-evaluate them in the future.
+
+=== Redis
+
+* https://ibm.github.io/operator-for-redis-cluster/
+** Very alpha
+* https://github.com/spotahome/redis-operator
+** Somewhat promising
+** Maybe in the future, for now
+** Evaluation by a third party: https://blog.palark.com/failure-with-redis-operator-and-redis-data-analysis-tools/
+* https://operatorhub.io/operator/redis-operator
+** No sentinel support
+
+Bitnami Helm Chart: https://github.com/bitnami/charts/tree/main/bitnami/redis/
+
+=== MariaDB/Galera
+
+* https://github.com/Orange-OpenSource/galera-operator
+** Archived
+* https://docs.percona.com/percona-operator-for-mysql/pxc/
+** Only supports MySQL and not MariaDB
+* https://github.com/cybozu-go/moco
+** Only supports MySQL and not MariaDB
+** Very promising concept though.
+Same principle as Patroni.
+
+Bitnami Helm Chart: https://github.com/bitnami/charts/tree/main/bitnami/mariadb-galera/
+
+== Decision
+
+Bitnami Helm Charts.
+
+Both Helm Charts can cover the single node use-case and the clustered use-case.
+
+Advantages::
+
+* Can easily be forked and maintained in case of breaking changes
+* Haven been used in SPKS for years now
+
+Disadvantages::
+
+* Breaking changes could make the maintenance harder
+
+=== Rationale
+
+Helm Charts are nowhere nearly as complicated as operators if the need for a fork arises.
```

Comment:
I am uncertain if this statement makes sense, e.g. we decided for an operator for PostgreSQL, why should that now be different?

Reply:
What @Kidswiss meant here is that having `our own operator` is costly in term of time and effort and we are better off using an existing (or forked) Helm Char.t

Reply:
Yeah, `our own` is a good keyword here.